core/src/main/kotlin/com/bridge/ouroboros/compose/ExecutableEffect.kt

```diff
@@ -23,4 +32,14 @@ abstract class ExecutableEffect<EVENT, EFFECT_STATE> : CoroutineScope {
             currentContext = null
         }
     }
-}
\ No newline at end of file
+
+    companion object {
+        @Volatile
+        var defaultDispatcher: CoroutineDispatcher = Dispatchers.Main
```

Comment:
Does this need to be public?

Reply:
as I see the usage, not really, it can be private